tasks/collectd.yml

```diff
@@ -13,10 +13,19 @@
   when: collectd_use_ppa
   tags: [collectd, collectd-install]
 
-- import_tasks: install.deb.yml
-  when: not collectd_use_ppa and ansible_os_family == 'Debian'
+- import_tasks: install.src.yml
+  when:
+    - not collectd_use_ppa
+    - ansible_os_family == 'Debian'
+    - ansible_lsb.codename != "bionic"
```

Comment:
Don't you want to do the oposite? 🤔 
If it's bionic, install from source and if not from package?

Reply:
No, I want exactly this. Bionic has a recent enough version in the universe repo, no need for PPA nor source install (which is shitty)